### PR TITLE
chore: Extra dependencies in setup.py for Go <-> Python interop

### PR DESCRIPTION
**Pull request by pyalex**
Signed-off-by: pyalex <moskalenko.alexey@gmail.com>

**What this PR does / why we need it**:

In order for Python to fully interop with Go we might need to install some extra python packages. This PR adds new dependency group `go` to `setup.py`.

**Which issue(s) this PR fixes**:

Fixes #

### CODE REVIEW

File: sdk/python/setup.py [hunk 1]
```diff
@@ -113,6 +113,10 @@
     "great_expectations>=0.14.0,<0.15.0"
 ]
 
+GO_REQUIRED = [
+    "cffi==1.15.0",
```
Comment (achals) [inline sdk/python/setup.py:117]: Is pinning in, or should we leave the patch version/minor version floating?

**Review by achals — COMMENTED** [review]
Can you add some context in the PR description?

File: sdk/python/setup.py (hunk 1 above)
Comment (pyalex) [reply sdk/python/setup.py]: Sure, keeping patch version floating

**Comment by achals** [comment]
Also, do we need to change any of our packaging scripts?

**Review by adchia — APPROVED** [review]
/lgtm

**Pull request merged**